src/Components/App/App.js

```diff
@@ -39,6 +39,7 @@ class App extends Component {
 
   showHome = () => {
     this.setState({ featuredFilm: null });
+    this.componentDidMount();
```

Comment:
Generally - you don't want to call componentDidMount directly - it's a lifecycle method that will automatically fire when the component is done mounting.